datalab/bigquery/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2015 Google Inc. All rights reserved.
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
Actually, the year listed here should be the year of the first version, not the latest.

I.E. we never change it for an existing file.